src/main/scala/strawman/collection/immutable/NumericRange.scala

```diff
@@ -169,25 +174,25 @@ final class NumericRange[T](
   //
   //   (0.1 to 0.3 by 0.1 contains 0.3) == true
   //
-//  private[immutable] def mapRange[A](fm: T => A)(implicit unum: Integral[A]): NumericRange[A] = {
-//    val self = this
-//
-//    // XXX This may be incomplete.
-//    new NumericRange[A](fm(start), fm(end), fm(step), isInclusive) {
-//
-//      private lazy val underlyingRange: NumericRange[T] = self
-//      override def foreach(f: A => Unit) { underlyingRange foreach (x => f(fm(x))) }
-//      override def isEmpty = underlyingRange.isEmpty
-//      override def apply(idx: Int): A = fm(underlyingRange(idx))
-//      override def containsTyped(el: A) = underlyingRange exists (x => fm(x) == el)
-//
-//      override def toString = {
-//        def simpleOf(x: Any): String = x.getClass.getName.split("\\.").last
-//        val stepped = simpleOf(underlyingRange.step)
-//        s"${super.toString} (using $underlyingRange of $stepped)"
-//      }
-//    }
-//  }
+  private[immutable] def mapRange[A](fm: T => A)(implicit unum: Integral[A]): NumericRange[A] = {
+    val self = this
+
+    // XXX This may be incomplete.
```

Comment:
I guess we need to add this case into collections-laws testing.  Can you make a note somewhere that mapRange-generated ranges should be laws-tested, in case I forget?